src/pages/steps/OrganizerStep.tsx

```diff
@@ -108,6 +204,60 @@ const OrganizerStep = ({
         }
         organizer={organizer}
       />
+      {isUitpasOrganizer && (
+        <Stack>
+          <Text fontWeight="bold" marginBottom={3}>
```

Comment:
```suggestion
        <Stack spacing={3}>
          <Text fontWeight="bold">
```

Instead of putting marginBottom on the children of the Stack, The Stack component has spacing which does this internally. 

On the other children too